### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by high0nc0cane**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by Jiahong-Guan — APPROVED** [review]
accept

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (Jiahong-Guan) [inline index.html]: adfwe

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (Jiahong-Guan) [inline index.html]: need good coding
```suggestion
here
```

File: index.html [hunk 3]
```diff
@@ -46,8 +46,8 @@ <h1>Tic Tac Toe</h1>
     <td><input type="checkbox" /></td>
   </tr>
 </table>
```
Comment (PandaMcGame) [inline index.html:48]:
```suggestion\
fgbdfbd
</table>
```
```suggestion
</table>
```
```suggestion
</table>
```

File: index.html [hunk 4]
```diff
@@ -46,8 +46,8 @@ <h1>Tic Tac Toe</h1>
...
-
-<h2 hidden>Game over</h2>
+here
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (Jiahong-Guan) [inline index.html]: gere
```suggestion
<h2 hidden>Game over, refresh to play in 🧑‍💻 🤖!</h2>
```

**Pull request merged**